intel_pytorch_extension_py/ops/pooling.py

```diff
@@ -25,7 +26,15 @@ def backward(ctx, grad_output):
 class MaxPoolingFunction(Function):
     @staticmethod
     def forward(ctx, input, kernel_size, stride, padding, dilation, ceil_mode):
-        output = core.max_pooling(input, (kernel_size,), (stride,), (padding,), (dilation,), ceil_mode)
+        if type(kernel_size) is int:
```

Comment:
`torch.nn.modules.utils._single` could serve the same purpose
```
from torch.nn.modules.utils import _single
```

Reply:
OK, thanks